AppCenterCrashes/AppCenterCrashes.xcodeproj/project.pbxproj

```diff
@@ -1720,6 +1741,13 @@
 /* End PBXProject section */
 
 /* Begin PBXReferenceProxy section */
+		AFB5AB72296461FC002EBC38 /* CrashReporter.framework */ = {
```

Comment:
Why was this added?

Reply:
hmm seems like this was a local config, will revert